src/hotspot/share/cds/archiveHeapLoader.cpp

```diff
@@ -78,10 +78,10 @@ void ArchiveHeapLoader::init_mapped_heap_relocation(ptrdiff_t delta, int dumptim
   _mapped_heap_relocation_initialized = true;
 }
 
-void ArchiveHeapLoader::init_narrow_oop_decoding(address base, int shift) {
+void ArchiveHeapLoader::init_narrow_oop_decoding(address dumptime_base, int shift) {
```

Comment:
AFAIU the first parameter is not really a dumptime base. It has different meaning depending on whether it is called for loaded or mapped heap regions. For loaded regions, it is dumptime base address, but for mapped regions, it represents the runtime base to use for decoding. So I would probably refrain from renaming it.